lib/src/signed_video_h26x_auth.c

```diff
@@ -747,7 +714,8 @@ validate_authenticity(signed_video_t *self)
       verify_success = verify_hashes_with_gop_hash(self, &num_expected_nalus, &num_received_nalus);
     }
   }
-
+  // Set |latest_validated_gop| to recived gop counter for the next validation.
+  self->gop_info->latest_validated_gop = self->gop_info->global_gop_counter;
```

Comment:
Isn't this in contradiction to line 654 where `latest_validated_gop` is incremented? If  a SEI is lost for example it looks like you first update from, for example, 4 to 6, and then increment on line 654 to 7.